lib/office365/rest/contact.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Office365
+  module REST
+    module Contact
+      def contact
```

Comment:
let's change the `contact` to `contacts`

Reply:
Done, rename all including the module name and file name.